app/models/saved_claim/disability_compensation.rb

```diff
@@ -1,17 +1,34 @@
 # frozen_string_literal: true
 
 class SavedClaim::DisabilityCompensation < SavedClaim
-  has_one(
-    :disability_compensation_submission,
-    class_name: 'DisabilityCompensationSubmission',
-    inverse_of: :disability_compensation_claim,
-    dependent: :destroy
-  )
-  has_one(
-    :async_transaction,
-    through: :disability_compensation_submission,
-    source: :disability_compensation_job
-  )
+  has_one :disability_compensation_submission,
+          class_name: 'DisabilityCompensationSubmission',
+          inverse_of: :disability_compensation_claim,
+          dependent: :destroy
+
+  has_one :async_transaction,
+          through: :disability_compensation_submission,
+          source: :disability_compensation_job
+
+  alias_attribute :submission, :disability_compensation_submission
 
   add_form_and_validation('21-526EZ')
+
+  attr_writer :form_hash
+
+  def self.from_hash(hash)
+    saved_claim = new(form: hash['form526'].to_json)
+    saved_claim.form_hash = hash
+    saved_claim
+  end
+
+  def to_submission_data(user)
+    {
+      'form_526' => EVSS::DisabilityCompensationForm::DataTranslation.new(
```

Comment:
The data translation returns `{ 'form_526' => { etc } }`, which will make this `'form_526' => { 'form_526' => {etc} }`. We'll either need to modify `DataTranslation.translate` to return the inner hash or use `merge` at the end of this hash to add it in. Personally I think modifying the data translation will be the more readable option.